dts/bindings/gpio/launchpad-header.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2019 Brett Witherspoon
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: LaunchPad GPIO header
+version: 0.1
+
+description: >
+    This is a representation of GPIO pin nodes exposed as headers on LaunchPads
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "launchpad-header"
+      generation: define
+
+    gpio-map:
+      type: compound
+      category: required
+
```

Comment:
I am a newbie when it comes to dts files. But looking at the arduino ones, should "#gpio-cells" be listed as a property here? The property is being used on line 10 of the dtsi files.

Reply:
LGTM otherwise

Reply:
I introduced `gpio-nexus.yaml` so including that like `arduino-header-r3.yaml` does will cover it.